src/Web/couch-potatoes-web-app/src/components/Search/SearchBar.tsx

```diff
@@ -0,0 +1,148 @@
+                                                                        alt={"poster of movie" + result.title}
+                                                                        borderRadius='lg'
+                                        />) : (<Image maxHeight={"80px"}
+                                                      maxWidth={"53px"}
+                                                      src={Background_Temp}
+                                                      alt={"no poster of movie" + result.title}
+                                                      borderRadius='lg'
+                                        />)}
+                                        <Flex alignItems={"center"}>
+                                            <Text>{result.title}</Text>
+                                        </Flex>
+
+                                    </Stack>
+
+                                </ListItem>
+
+                            ))}
+                            {searchResults?.people?.map((result) => (
+                                <ListItem padding={"2px"} bg={"white"} key={result.id} onClick={() =>
+                                    navigate(
+                                        `/Couch_Potatoes/person/${result.id}`
+                                    )
+                                }>
+                                    <Stack
+                                        direction={"row"} overflowX={"auto"} _hover={{cursor: "pointer"}}>
+                                        {(result?.profilePath) ? (<Image maxHeight={"80px"}
+                                                                        src={getPosterImageUri(result?.profilePath)}
+                                                                        alt={"poster of person" + result.name}
+                                                                        borderRadius='lg'
+                                        />) : (<Image maxHeight={"80px"}
+                                                      maxWidth={"53px"}
+                                                      src={Background_Temp}
+                                                      alt={"no poster of person" + result.name}
+                                                      borderRadius='lg'
+                                        />)}
+                                        <Flex alignItems={"center"}>
+                                            <Stack direction={"column"}>
+                                                <Text>{result.name}</Text>
+                                                <Text color={"grey"} fontSize={"sm"} fontStyle={"italic"}>Known for: {result.knownFor}</Text>
+                                            </Stack>
+
+                                        </Flex>
+
+                                    </Stack>
+
+                                </ListItem>
+
+                            ))}
+                        </List>) : null}
```

Comment:
For scenarios where you want to do the following conditional rendering, you can also do: 
```javascript
{
  someCondition && <MyComponent&> 
}
```

 Instead of 
 
 ```javascript
{
  someCondition ? <MyComponent&> : null
}
```

Reply:
fixed